packages/cli/src/util/lockfile.ts

```diff
@@ -1,37 +1,45 @@
-export type Lockfile = {
-  lockSync(path: string): void;
-  unlockSync(path: string): void;
-};
-
-const lockFile: Lockfile = (await import("lockfile")) as Lockfile;
-
-function getLockFilepath(filepath: string): string {
-  return `${filepath}.lock`;
```

Comment:
No longer required as proper-lockfile adds this extension by default, see [lib/lockfile.js#L12](https://github.com/moxystudio/node-proper-lockfile/blob/9f8c303c91998e8404a911dc11c54029812bca69/lib/lockfile.js#L12)